server/game/cards/events/07/scalingthewall.js

```diff
@@ -0,0 +1,50 @@
+const DrawCard = require('../../../drawcard.js');
+
+class ScalingTheWall extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: (event, challenge) => (
+                    challenge.winner === this.controller &&
+                    challenge.attackingPlayer === this.controller &&
+                    this.hasAttackingWildling()
+                )
+            },
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
+                    cardCondition: card => this.cardCondition(card),
+                    activePromptTitle: 'Select location',
+                    source: this,
+                    onSelect: (player, card) => this.onCardSelected(player, card)
+                });
+            }
+        });
+    }
+
+    hasAttackingWildling() {
+        let cards = this.controller.filterCardsInPlay(card => {
```

Comment:
Should be simpler to return the result of `this.controller.anyCardsInPlay` instead of filtering cards and then checking length (slowly trying to get rid of usages like this)